website/docs/components/code-block/partials/code/how-to-use.md

```diff
@@ -0,0 +1,148 @@
+## How to use this component
+
+The basic invocation requires `@value` to be passed.
+
+```handlebars
+<Hds::CodeBlock
+  @value="aws ec2 --region us-west-1 accept-vpc-peering-connection"
+/>
+```
+
+### Title and description
+
+Optionally, you can pass a title or a description.
```

Comment:
Probably, for sake of simplicity/brevity, I would do a single example with both and change the paragraph this way:
```suggestion
Optionally, you can pass a title and/or a description.
```

Reply:
Addressed in [4ea291b](https://github.com/hashicorp/design-system/pull/1749/commits/4ea291bbca9846b2dd2d1bbe4e6151c6fca2e066)